doc/api/http.md

```diff
@@ -2755,6 +2755,16 @@ http.get('http://nodejs.org/dist/index.json', (res) => {
 }).on('error', (e) => {
   console.error(`Got error: ${e.message}`);
 });
+
+// Create a local server to receive data from
+const server = http.createServer((req, res) => {
+  res.writeHead(200, { 'content-type': 'application/json' });
+  res.end(`{
+    'data': 'hello world'
+  }`);
```

Comment:
That's not valid JSON, you need to use double quote:
```suggestion
  res.end(`{
    "data": "Hello World!"
  }`);
```

Alternatively we can use `JSON.stringify`:
```suggestion
  res.end(JSON.stringify({
    data: 'Hello World!'
  }));
```